src/main/java/core/basesyntax/actions/DoingAction.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.actions;
```

Comment:
same for the other handlers
```suggestion
package core.basesyntax.strategy.handler;
```